Podcast/Episode.swift

```diff
@@ -58,29 +58,47 @@ class Episode: NSObject {
      convenience init(json: JSON) {
         let id = json["id"].intValue
         let title = json["title"].stringValue
-        let dateString = json["date"].stringValue
-        let descriptionText = json["description"].stringValue
+        let dateString = json["pubDate"].stringValue
+        let descriptionText = json["summary"].stringValue
         let isRecommended = json["is_recommended"].boolValue
         let isBookmarked = json["is_bookmarked"].boolValue
         let numberOfRecommendations = json["n_recommendations"].intValue
-        let seriesTitle = json["series_title"].stringValue
-        let duration = json["duration"].doubleValue
+        let seriesTitle = json["seriesTitle"].stringValue
+        let seriesId = json["seriesId"].intValue
+        let duration = json["duration"].stringValue
         let tags = json["tags"].arrayValue.map({ (tag: JSON) in Tag(name: tag.stringValue) })
-        let audioURL = URL(string: json["audioURL"].stringValue)
+        let audioURL = URL(string: json["audioUrl"].stringValue)
         let dateCreated = DateFormatter.parsingDateFormatter.date(from: dateString) ?? Date()
-        let smallArtworkURL = URL(string: json["small_image_url"].stringValue)
-        let largeArtworkURL = URL(string: json["large_image_url"].stringValue)
+        let smallArtworkURL = URL(string: json["imageUrlSm"].stringValue)
+        let largeArtworkURL = URL(string: json["imageUrlLg"].stringValue)
         
-        self.init(id: id, title: title, dateCreated: dateCreated, descriptionText: descriptionText, smallArtworkImageURL: smallArtworkURL, series: nil, largeArtworkImageURL: largeArtworkURL, audioURL: audioURL, duration: duration, seriesTitle: seriesTitle, tags: tags, numberOfRecommendations: numberOfRecommendations, isRecommended: isRecommended, isBookmarked: isBookmarked)
+        self.init(id: id, title: title, dateCreated: dateCreated, descriptionText: descriptionText, smallArtworkImageURL: smallArtworkURL, seriesId: seriesId, largeArtworkImageURL: largeArtworkURL, audioURL: audioURL, duration: duration, seriesTitle: seriesTitle, tags: tags, numberOfRecommendations: numberOfRecommendations, isRecommended: isRecommended, isBookmarked: isBookmarked)
      }
     
     // Returns data - time - series in a string
     func dateTimeSeriesString() -> String {
         let dateFormatter = DateFormatter()
         dateFormatter.dateStyle = .long
         dateFormatter.timeStyle = .none
+        let length = duration
+        
+        //add back in later when we figure out what the data is consistently
```

Comment:
No because I was talking to joe and he plans to standardize the data more on his end and rn some use hh:mm:ss, mm:ss, hh:mm, etc .. this will be fixed later on once we get a standard format